Assets/Fungus/Scripts/Editor/FlowchartWindow.cs

```diff
@@ -1205,7 +1125,7 @@ protected virtual void DrawConnections(Flowchart flowchart, Block block, bool hi
 
             var connectedBlocks = new List<Block>();
 
-            bool blockIsSelected = (flowchart.SelectedBlock != block);
+            bool blockIsSelected = flowchart.SelectedBlock == block;
```

Comment:
This was making it so no connections were highlighted. I think it's been like that for a while. Seems fine after the change.